src/components/edit/block-dnd/ChordTokenDroppable.tsx

```diff
@@ -1,11 +1,11 @@
-import { RootRef } from "@material-ui/core";
 import React, { useEffect } from "react";
+import { ConnectDropTarget } from "react-dnd";
 import { ChordBlock } from "../../../common/ChordModel/ChordBlock";
 import { IDable } from "../../../common/ChordModel/Collection";
 import { useChordDrop } from "./useChordDrop";
 
 interface ChordTokenDroppableProps {
-    children: React.ReactElement;
+    children: (dropRef: ConnectDropTarget) => React.ReactElement;
```

Comment:
opted for this sort of function child pattern since it's not a standard ref, so React.cloneElement on children with the ref seemed a bit dangerous, since it's not the most typesafe operation. this way at least there's typechecking